### ISSUE TITLE
New versions of keyrings may require keyrings.alt

### ISSUE THREAD

**Issue by davidderus** [davidderus/pydebrid#5]
```
Python-Keyring is a module that provides a cross-platform interface to store passwords, and has multiple backends.

Recently that package has been split into two parts: base (python-keyring) which contains only recommended secure backends, and keyrings.alt which contains all the other backends. This ITP is for the latter package. I am going to make python-keyring recommend keyrings.alt
```

That's a system-specific issue.

**Comment by davidderus** [comment]
Two solutions:
- Adding keyrings.alt to requirements thus keeping keyrings up to date.

Or
- Setting keyrings to a version before the split.

**Comment by FrankenLab** [comment]
I think the first solution might be better, if you set it to a version before the split, then what happens when that version becomes obsolete?

Am I missing something?

**Comment by davidderus** [comment]
No you're not, that's indeed the safer solution, but need some tests in order to handle possible API changes.

The second one is a lazy fix, and not a good one ;)

**Comment by davidderus** [comment]
Fixed in 1.0.8 release, now closing the issue.
Feel free to reopen it if needed.
